### DISCUSSION TITLE
Student Developer Pack Not Available

### DISCUSSION THREAD

**Discussion by CRISvsGAME** [community/community#120285] [GitHub Education] [answered]
### Hi everyone,

It is worth mentioning that the re-validation process has some issues as I had to try the submission multiple times until it was successfully sent. It has been one week and the Student Developer Pack benefits are still not available.

![image](https://github.com/community/community/assets/101801362/0b25103d-ed82-4f7a-a87e-150ef1d152ae)

**Comment by fahadshahbaz** [comment]
Lots of people applying can make it take longer to get your Student Developer Pack benefits. Just be patient. If you're still having trouble, you can ask support for help.

**Comment by CRISvsGAME** [comment]
Thank you @fahadshahbaz for taking the time to answer.
I initiated this discussion after thorough research and based on my personal experiences. I had already highlighted the issues when I submitted my post. Community members have expressed concerns about being approved but not receiving the benefits.
I believe there should be an official statement clarifying that the process might take days, weeks, or months, instead of using vague phrases like "your benefits will be available soon" or "your benefits are now available," which can be misleading or incorrect.

**Comment by fahadshahbaz** [comment]
It's frustrating to be in a situation where you're waiting without any clear indication of how long the wait will be 😑.

**Comment by CRISvsGAME** [comment, marked as answer]
It is likely just a matter of time until the benefits are activated. In my case, it took around two weeks. Thank you.
